src/car.hpp

```diff
@@ -1,22 +1,16 @@
 #pragma once
 
 #include "common.hpp"
+#include "direction.hpp"
 #include <string>
 
 using std::string;
 
-enum class direction : int

```

Comment:
Do we need to have it as a separate class? We could have it in lane class but as 
enum class direction {north , east, south, west,}; 
will already have north = 0, east = 1, south = 2, west = 3. 

https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum

Reply:
It's still just an enum class instead of a regular class, it's just in a separate file now for clarity. It doesn't even need to be compiled, just #included where necessary.

I moved it to a separate file because multiple classes (Car and LaneManager) are using it now. Even if we expand it to include more directions, both those classes will still need to use it (I think we will have to pass it to cars when we construct them so they know which direction to drive into the screen from). I suppose we could just define it in `lane_manager.hpp` or whatever and just make sure to include that, but I feel like that's more confusing than having it in its own file.

Reply:
Yup, that makes sense! Will leave it as is.